vm/state_manager/src/errors.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use amt::Error as AmtErr;
+use std::fmt;
+
+#[derive(Debug, PartialEq)]
+pub enum Error {
+    /// Error orginating from state
+    State(String),
+    /// Error originating from amt
+    AMT(String),
```

Comment:
Whats the difference between these? Aren't AMT errors also state errors?

Reply:
State error originates from `state_manager`, whereas, AMT originates from the AMT. Happy to consolidate if that's the group's consensus.

Reply:
I feel like the distinction would be lost on most people, we also don't really expect AMT errors as they would probably be fatal.

Reply:
Yeah that makes sense to me! Will change.